### PR TITLE
Add FreeBSD 12.2-RELEASE AMI

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Add FreeBSD 12.2-RELEASE AMI

### CODE REVIEW

**Review by Akm0d — DISMISSED** [review]

**Comment by bryceml** [comment]
where did this ami come from? It doesn't look like we've promoted a freebsd 12.2 image yet.

**Comment by krionbsd** [comment]
> where did this ami come from? It doesn't look like we've promoted a freebsd 12.2 image yet.

https://lists.freebsd.org/pipermail/freebsd-announce/2020-October/001993.html

File: cicd/jenkins/kitchen-freebsd122-py3 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@Library('salt@master-1.9') _
+
+runTestSuite(
+    ami_image_id: 'ami-0c273f14bc9df57d1',
```
Comment (bryceml) [inline cicd/jenkins/kitchen-freebsd122-py3:4]: This ami should come from promoting https://github.com/saltstack/salt-jenkins/pull/1682 after it has been merged.

Comment (krionbsd) [reply cicd/jenkins/kitchen-freebsd122-py3:4]: yeah, that's correct, so I created 2 PRs which should be merged in series

**Review by bryceml — APPROVED** [review]

**Pull request merged**